.github/workflows/tests.yaml

```diff
@@ -105,7 +105,7 @@ jobs:
           fetch-depth: 0
       - name: Set up uv with caching
         id: setup-uv
-        uses: astral-sh/setup-uv@v4
+        uses: astral-sh/setup-uv@v5
         with:
           enable-cache: true
```

Comment:
Can you drop this now that it is true by default?

Reply:
I'll do that separately.